src/verilog.y

```diff
@@ -2460,6 +2482,11 @@ cellpinItemE<pinp>:		// IEEE: named_port_connection + empty
 //************************************************
 // EventControl lists
 
+attr_event_controlE<sentreep>:
+		/* empty */			{ $$ = NULL; }
+	|	attr_event_control	{ $$ = $1; }
```

Comment:
For all insertions please tab the {'s out to the standard 7th tab column.
For really long lines the { goes on a new line with 3 tabs.